tests/rigswr.c

```diff
@@ -346,7 +346,7 @@ static int set_conf(RIG *rig, char *conf_parms)
 
         if (!q)
         {
-            return RIG_EINVAL;
```

Comment:
This change is ok because the code is parsing a line was supposed to contain an equal sign but it is not.